forge-game/src/main/java/forge/game/Game.java

```diff
@@ -860,11 +866,23 @@ public void onPlayerLost(Player p) {
         // 901.6: If the current planar controller would leave the game, instead the next player
         // in turn order that wouldn't leave the game becomes the planar controller, then the old
         // planar controller leaves
+        if (planarControllerLost) {
+            for (Card c : getActivePlanes()) {
+                if ((c != null) && !(c.getOwner().equals(p))) {
+                    c.setController(getNextPlayerAfter(p), 0);
+                    getAction().controllerChangeZoneCorrection(c);
```

Comment:
if you want try to also check for
> 901.10b. Abilities from phenomena owned by a player who left the game remain on the stack controlled by the new planar controller.
inside `removeInstancesControlledBy`?
Hopefully the engine can tolerate the card being gone